content/graph/MinCostMaxFlow.h

```diff
@@ -2,36 +2,35 @@
  * Author: Stanford
  * Date: Unknown
  * Source: Stanford Notebook
- * Description: Min-cost max-flow. cap[i][j] != cap[j][i] is allowed; double edges are not.
+ * Description: Min-cost max-flow.
  *  If costs can be negative, call setpi before maxflow, but note that negative cost cycles are not supported.
  *  To obtain the actual flow, look at positive values only.
  * Status: Tested on kattis:mincostmaxflow, stress-tested against another implementation
- * Time: Approximately O(E^2)
+ * Time: $O(F \cdot E \log(V))$ where F is max flow. $O(VE)$ for setpi.
```

Comment:
```suggestion
 * Time: $O(F E \log(V))$ where F is max flow. $O(VE)$ for setpi.
```
to avoid text overlap in pdf